### PR TITLE
Migrate some of our Lambdas into the monitoring stack

### PR DESCRIPTION
**Pull request by alexwlchan**
### What is this PR trying to achieve?

Migrate specific Lambdas into the monitoring stack. At least initially:

* post_to_slack
* gatling_to_cloudwatch
* update_service_list

### Who is this change for?

A platform team who want stacks that make sense.

Part of #980.

### Have the following been considered/are they needed?

- [x] Run `terraform apply`.

### CODE REVIEW

**Comment by alexwlchan** [comment]
This is applied in both stacks, so I’d like to merge this soon before it gets blatted by something else.

File: builds/publish_lambda_zip.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def build_lambda_local(path, name):
 
     # Copy all the associated files to the Lambda directory.
     for f in os.listdir(path):
-        if not f.startswith(('test_', '.', 'requirements.txt')):
+        if not f.startswith(('test_', '.', 'requirements.txt', '__pycache__')):
```
Comment (kenoir) [inline builds/publish_lambda_zip.py]: 👍

**Comment by kenoir** [comment]
Not just lambdas in here, but still good 👍

**Comment by kenoir** [comment]
Looks like there's a failing test in `master` for the `id_minter` that wasn't introduced by these changes (in addition to the lambda tests that are currently re-running. Good to merge if the lambda tests pass - and we'll address the `id_minter` later.

**Pull request merged**